src/parser/parser.h

```diff
@@ -134,7 +134,7 @@ inline std::ostream& operator<<(std::ostream& out, SymbolType type) {
  */
 class CVC4_PUBLIC Parser {
   friend class ParserBuilder;
-
+protected:
```

Comment:
We should strongly prefer keeping implementation details private. Why is this change being made?

Reply:
I refactored so that this is no longer necessary (I needed access to d_exprManager in smt2.cpp but realized getExprManager() is used throughout).